### PR TITLE
[DPEDE-363] Migrate utilities sizing to new Vue documentation

### PR DESCRIPTION
**Pull request by kosgor**

### CODE REVIEW

File: src/documentation/pages/utilities-wip/sizing/examples/height.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template lang="pug">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/utilities-wip/sizing/examples/height.vue:1]: Rename to _height.vue

File: src/documentation/pages/utilities-wip/sizing/examples/maximum-width-height.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<template lang="pug">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/utilities-wip/sizing/examples/maximum-width-height.vue:1]: Rename to _maximum-width-height.vue

File: src/documentation/pages/utilities-wip/sizing/examples/width.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<template lang="pug">
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/utilities-wip/sizing/examples/width.vue:1]: Rename to _width.vue

File: src/documentation/pages/utilities-wip/sizing/index.vue [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import Examples from './examples/index.vue';
+
+@Component({
+  head() {
+    return {
+      title: 'Sizing - Color'
```
Comment (josecarlosgonzalezv) [inline src/documentation/pages/utilities-wip/sizing/index.vue:14]:
```suggestion
      title: 'Chi - Sizing'
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

**Review by jllr — APPROVED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**